micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/login.js

```
@@ -140,23 +141,30 @@
   const { mode } = Digit.Hooks.useQueryParams();
-  if (mode === "admin" && config?.[0]?.body?.[2]?.disable == false && config?.[0]?.body?.[2]?.populators?.defaultValue == undefined) {
+  if (
+    mode === "admin" && config?.[0]?.body?.[2]?.disable == false && config?.[0]?.body?.[2]?.populators?.defaultValue == undefined ) {
     config[0].body[2].disable = true;
     config[0].body[2].isMandatory = false;
     config[0].body[2].populators.defaultValue = defaultValue;
   }
-
+  const defaultValues = config[0].body.reduce((acc, curr) => {
+    if (curr?.populators?.defaultValue) {
+      acc = { ...acc, [curr?.populators?.name]: curr?.populators?.defaultValue };
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)